front/src/components/App.tsx

```diff
@@ -42,17 +43,25 @@ const App = ({ isLoggedIn, fetchfollowedShows }: Props) => {
 
         <Switch>
           <Route path="/" exact>
-            <SearchPage />
+            <ErrorBoundary>
+              <SearchPage />
+            </ErrorBoundary>
```

Comment:
@zeaus Would it work to have one ErrorBoundary around all the routes? Or do we need to repeat it inside every route?

Reply:
@trybick I added it against every route so that it keeps the route working, but I checked it now that it replaces the whole tree and the routes stop working anyway so one ErrorBoundary around all the routes would be the way to go. 

Link: https://stackoverflow.com/questions/57658813/why-does-routing-stop-working-even-when-using-an-errorboundary

Reply:
I found two other ideas for this:

- Add key to the ErrorBoundary
https://stackoverflow.com/a/61415043
- Use higher order component withRouter
https://stackoverflow.com/a/49341596

If you want see if either will work, feel free. Otherwise I can merge this in a few days and check myself :)

Reply:
I implemented the solution using withRouter.
https://stackoverflow.com/questions/48121750/browser-navigation-broken-by-use-of-react-error-boundaries/49341596#49341596

It is working now and the code has been pushed as well.